### PR TITLE
standardize yolov8backbone

### PR DESCRIPTION
**Pull request by haifeng-jin**

### CODE REVIEW

File: keras_cv/models/object_detection/yolo_v8/yolo_v8_backbone.py [hunk 1]
```diff
@@ -71,23 +72,65 @@ def apply_spatial_pyramid_pooling_fast(
 # TODO(ianstenbit) This should probably just be a CSPDarknet
 @keras.utils.register_keras_serializable(package="keras_cv.models")
 class YOLOV8Backbone(Backbone):
+    """Implements the YOLOV8 backbone for object detection.
```
Comment (jbischof) [inline keras_cv/models/object_detection/yolo_v8/yolo_v8_backbone.py:75]: You can add a note that it is a variant of the CSPDarknet architecture.

File: keras_cv/models/object_detection/yolo_v8/yolo_v8_backbone.py [hunk 2]
```diff
@@ -71,23 +72,65 @@ def apply_spatial_pyramid_pooling_fast(
...
+    [guide to transfer learning & fine-tuning](https://keras.io/guides/transfer_learning/).
+
+    Args:
+        stackwise_channels: A list of ints, the number of channels for each dark
+            level in the model.
+        stackwise_depth: A list of ints, the depth for each dark level in the
+            model.
+        include_rescaling: bool, whether to rescale the inputs. If set to
+            True, inputs will be passed through a `Rescaling(1/255.0)` layer.
+        activation: String. The activation functions to use in the backbone.
```
Comment (jbischof) [inline keras_cv/models/object_detection/yolo_v8/yolo_v8_backbone.py:87]: "to use in the CSPDarkNet blocks"

**Review by ianstenbit — APPROVED** [review]
LGTM -- thanks Haifeng!

**Review by ianstenbit — APPROVED** [review]
LGTM -- thanks Haifeng!

**Review by jbischof — APPROVED** [review]
Looks great, thank you!

**Pull request merged**